app/javascript/react/components/RestaurantIndex.js

```diff
@@ -0,0 +1,36 @@
+
+import React, { useEffect, useState } from 'react'
+import RestaurantTile from './RestaurantTile'
+
+const RestaurantIndex = (props) => {
+  const [restaurantData, setRestaurantData] = useState([])
+
+   useEffect(() => {
+       fetch('/api/v1/restaurants?location=boston')
+             .then(response => response.json())
+             .then(responseBody => {
+                 setRestaurantData(responseBody)
+             })
+      }, [])
+
+  const restaurantTileArray  = restaurantData.map((restaurant) => {
+      
+        return(
+            <div>
+                <RestaurantTile
```

Comment:
Indentation in this file looks to be about 4 spaces whereas other files are 2, it's best practice to have this match across files in a project